configuration/items.md

```diff
@@ -13,7 +13,7 @@ While a device or service might be quite specific, Items are unified substitutio
 Items can be Strings, Numbers, Switches or one of a few other basic Item types, a programmer can rightly compare Item Types with base variable types of a programming language.
```

Comment:
@bgilmer77 this would be the best way to comment on a part of this PR. You simply click on "Files changed", then click the + symbol left to a line changed and add your comment. see:  https://user-images.githubusercontent.com/2870104/30526250-c4ec64b6-9c17-11e7-8bf7-ea0b0af234e6.png